compiler-rt/lib/radsan/radsan_context.h

```diff
@@ -10,15 +10,10 @@
 
 #include <radsan/radsan_user_interface.h>
 
-#include <functional>
-
 namespace radsan {
 
 class Context {
 public:
-  Context();
-  Context(std::function<OnErrorAction()> get_error_action);
```

Comment:
This seemed to me like a way to do dependency injection. If we want to keep the ability to do something like that, I could introduce a `bool shouldExitOverride` or similar that would allow the unit test to inject it's own way of handling the error?

See the reworked unit test for more info